tests/integration/test_manipulate_statistics/test.py

```diff
@@ -122,3 +126,32 @@ def test_single_node_normal(started_cluster):
     """
     )
     run_test_single_node(started_cluster)
+
+
+def test_replicated_table_ddl(started_cluster):
```

Comment:
Maybe create a tests where you modify the columns (e.g.: rename, change type to a valid one, change type to invalid type, drop column, etc) and verify that CH can handle it without issues.

Reply:
Goed idee! on it (actually I found new bugs :(

Reply:
done, PTAL @antaljanosbenjamin